src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/12864128/b600451c-deb6-4f3d-8e05-00b505c9e124)

Reply:
How should I do that?

Reply:
Done.